src/js/draad-maps.js

```diff
@@ -202,27 +218,35 @@ class Draad_Map {
 
 		marker.locationTrap = new Draad_Focus_Trap(location);
 
-		this._markerHandler( marker, location );
+		this.markerHandler( marker, location );
 
 		return marker;
 
 	}
 
-	_markerHandler = ( marker, location = null, markerSrc = null, markerActiveSrc = null ) => {
+	/**
+	 * Event handler for markers.
+	 * 
+	 * @param {object} marker The marker object.
+	 * @param {HTMLElement} location The infowindow.
+	 * @param {string} markerSrc The default marker image.
+	 * @param {string} markerActiveSrc The active marker image.
+	 */
+	markerHandler = ( marker, location = null, markerSrc = null, markerActiveSrc = null ) => {
 
 		const draad = this;
```

Comment:
Is dit wel nodig? Want als je arrow functions gebruikt (zoals je hier doet) dan kun je gewoon `this` gebruiken, toch?